src/Cake.Web/App_Data/addins.xml

```diff
@@ -539,4 +539,14 @@
     <Description>Cake Addin to allow message posting to HipChat.</Description>
     <Categories>Communication</Categories>
   </Addin>
+  <Addin>
+    <Name>Cake.DoInDirectory</Name>
+    <NuGet Id="Cake.DoInDirectory">
+      <Filter>/**/Cake.DoInDirectory.dll</Filter>
```

Comment:
@pitermarx in order to generate the documentation on the site, we are going to need the associated XML file that can be generated as part of the build process.  This can be done by enabling that setting within the project properties.  Let me know if you are not sure how to do this.